src/utilities/stringToCmdLine.cpp

```diff
@@ -16,7 +16,7 @@
 
 namespace utilities
 {
-StringToCmdLine::StringToCmdLine(const std::string &cmdString)
+StringToCmdLine::StringToCmdLine(const std::string &cmdString) : argCount(0)
```

Comment:
this initialization should be in the header, not the cpp file

Reply:
Right, I'll move it. Is this something there's a tidy warning for, in general?